core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -254,6 +265,76 @@ class UserService(
       RawlsBillingProjectResponse(roles, project.copy(message = message, status = CreationStatuses.Error))
   }
 
+  def updateLandingZoneStatus(billingProject: Option[RawlsBillingProject]): Future[Option[RawlsBillingProject]] =
+    billingProject match {
+      case None    => Future.successful(billingProject)
```

Comment:
I don't quite understand… how would we get into this with billingProject being None?

Reply:
`getBillingProject` uses a method that returns an optional
`updateLandingZoneStatus` returns a future now
Every way I tried, the `<-` in `getBillingProject` kept getting interpreted as the .map of the optional, not the future
I could make it work with some really convoluted syntax, but overloading the function like this was the least confusing way I could find to implement it